lib/data_update_scripts/20200821103405_remove_orphaned_credits_by_organization.rb

```diff
@@ -0,0 +1,23 @@
+module DataUpdateScripts
+  class RemoveOrphanedCreditsByOrganization
+    def run
+      # Apparently we have a bunch of Credits that don't belong to either user or org
+      ActiveRecord::Base.connection.execute(
+        <<~SQL,
+          DELETE FROM credits
```

Comment:
Is this because users/orgs were banned or are these orphaned credits a result of something else or no idea? 🙃

Reply:
I think they were just created for test: https://dev.to/admin/blazer/queries/199-orphan-credits - They don't have a cost nor a user nor an org. Probably before validations were added